test/models/amounts.spec.ts

```diff
@@ -5,7 +5,7 @@ import {
 } from "../../src/amounts";
 import { Fraction } from "../../src/fraction";
 import BN from "bn.js";
-import { assert } from "chai";
+import assert from "assert";
```

Comment:
Unit tests should continue to use `chai` since it is a more powerful assertion library. Just the code in `src/` shouldn't.